internal/kafka/command_link_task_list.go

```diff
@@ -75,24 +86,36 @@ func (c *linkCommand) taskList(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	tasks := make([]task, len(link.GetTasks()))
+	for i, t := range link.GetTasks() {
+		errs := make([]taskErr, len(t.Errors))
+		for j, e := range t.Errors {
```

Comment:
```suggestion
	for i, task := range link.GetTasks() {
		errors := make([]taskErr, len(t.Errors))
		for j, error := range t.Errors {
```

Reply:
I can't use `task` for the variable name apparently because there is already the `task` struct. Is there any other best practice here?

Reply:
:( rename the `task` struct or leave it as is

Reply:
I could rename the struct to `taskDescription`?

Reply:
I'll just leave as is - doesn't seem like a big thing and it's not customer facing so we can change it in the future.